master/docs/manual/cfg-reporters.rst

```diff
@@ -879,6 +879,39 @@ As a result, we recommend you use https in your base_url rather than http.
     :param string password: the stash user's password
     :param list builders: only send update for specified builders
 
+.. bb:reporter::BitbucketStatusPush
+
+BitbucketStatusPush
+~~~~~~~~~~~~~~~~~~~
+
+.. @cindex BitbucketStatusPush
+.. py:class:: buildbot.reporters.bitbucket.BitbucketStatusPush
+
+::
+
+    from buildbot.plugins import reporters
+    bs = reporters.BitbucketStatusPush('oauth_key', 'oauth_secret')
```

Comment:
this will not work unless you update `master/setup.py` file: all plugins are listed there